entries/p/paysafecard.com.json

```
@@ -3,9 +3,13 @@
     "domain": "paysafecard.com",
     "url": "https://www.paysafecard.com/",
     "tfa": [
-      "totp"
+      "sms",
+      "custom-software"
     ],
-    "documentation": "https://www.paysafecard.com/en-au/lp-products/2-step-login/",
+    "custom-software": [
+      "paysafecard app"
+    ],
+    "documentation": "https://www.paysafecard.com/en/help/2-faktor-authentifizierung/",
```

As you mention 2FA is mandatory and the page you linked to only covers recovery I think the `recovery`-element is a better fit.
```suggestion
    "recovery": "https://www.paysafecard.com/en/help/2-faktor-authentifizierung/",
```